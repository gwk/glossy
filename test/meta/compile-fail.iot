{
  'cmd': '$PROJ/iotest.py -no-times test-meta/$NAME.iot',
  'links' : {'test-meta'},
  'code': 1,
}
