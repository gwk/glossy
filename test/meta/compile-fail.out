test-meta/compile-fail
compile process was expected to return code: 0; actual code: 1.

compile step 0 failed: `test-meta/compile-fail.py`
[30mcat ../../../test-meta/compile-fail/compile-fail.compile-out-00[0m
[31mcompile-fail: std out.[0m
[30mcat ../../../test-meta/compile-fail/compile-fail.compile-err-00[0m
[31mcompile-fail: error.[0m
================================================================


################################################################
TESTS FOUND: 1; BROKEN: 0; SKIPPED: 0; FAILED: 1.
